CHANGELOG.md

```diff
@@ -1,3 +1,14 @@
+## 1.7.6 - 2018-02-14
+This is a bugfix release.
+
+Bugfix:
+  * Reverts commit setting `build_tar: FALSE` in foss build defaults. This fix
+    was masking a bug in github.com/puppetlabs/packaging which has since been
+    fixed.
+  * (CPR-515) It was not previously possible to specify that postinstall tasks
```

Comment:
I was torn for this one. It's technically a new feature in ezbake, but was added for a bugfix. 🤷‍♀️

Reply:
Does this change affect how people would be using ezbake? Like, will people have to change what param names they use in order to maintain old functionality? I forget the specifics of the change, but I assume not, so I'm cool calling this a bugfix.

Reply:
Nothing they need to change, only if they want something only to be run on install. Let's go with bugfix

Reply:
You don't have to change anything to maintain existing functionality, but, say, to fix the issue reported against puppetserver you will need to change you ezbake config (whichever tasks should only run on install will need to be moved from the `postinst` section https://github.com/puppetlabs/puppetserver/blob/5.1.x/resources/ext/ezbake.conf#L23-L35 to a `postinst-install` section.

Reply:
Hmmm what's the definition of a feature? Added functionality?

Reply:
from semver.org:
> Given a version number MAJOR.MINOR.PATCH, increment the:
>
> MAJOR version when you make incompatible API changes,
> MINOR version when you add functionality in a backwards-compatible manner, and
> PATCH version when you make backwards-compatible bug fixes.

Reply:
starting to think this is a feature release then, but I don't have super strong feelings

Reply:
Just call it a bugture